changelogs/fragments/60510-k8s-apply-check-mode.yml

```
@@ -0,0 +1,2 @@
+minor_bugfixes:
```

```suggestion
bugfixes:
```
There's `bugfixes` and `minor_changes`, but no `minor_bugfixes`.